packages/react-signer/src/Modal.tsx

```diff
@@ -212,17 +225,11 @@ class Signer extends React.PureComponent<Props, State> {
                       ? t('Sign via {{hardwareType}}', { replace: { hardwareType: hardwareType || 'hardware' } })
                       : isExternal
                         ? t('Sign via Qr')
-                        : t('Sign and Submit')
-              }
-              icon={
-                isQrVisible
-                  ? 'qrcode'
-                  : currentItem.isUnsigned
-                    ? 'sign-in'
-                    : isExternal
-                      ? 'qrcode'
-                      : 'sign-in'
+                        : currentItem.isSign
```

Comment:
So here would have the "sign (no submission)" button, along with the others - instead of sending this through via the queues. So you can basically "sign & send" or "only sign" on any tx.